src/index.js

```diff
@@ -1 +1,68 @@
-import './style.css';
\ No newline at end of file
+import urlApi from './Constance.js';
+import './style.css';
+
+const URL = urlApi();
+const form = document.querySelector('form');
+const name = document.querySelector('input[type="text"]');
+const score = document.querySelector('input[type="number"]');
+const p = document.querySelector('.square');
+const btn = document.querySelector('.btn');
+const gameData = {
+  id: '',
+};
+
+const update = async () => {
+  await fetch(`${URL}games/${gameData.id}/scores/`, {
+    method: 'GET',
+    headers: { 'Content-type': 'application/json; charset=UTF-8' },
+  })
+    .then((response) => response.json())
+    .then((data) => {
+      // console.log('url:', `${URL}games/${gameData.id}/scores/`);
+      // console.log(data);
```

Comment:
- [ ] OPTIONAL
There are comments found in your code on the repository. Kindly endeavor to remove **all** comments from your code before pushing it to the repository so as to keep your code cleaner.